src/App.js

```diff
@@ -29,7 +29,7 @@ function App() {
 
       <nav className="navbar navbar-light bg-light static-top">
         <div className="container">
-          <a className="navbar-brand" href="#">The NAME - Heritage - Project</a>
```

Comment:
Hey, I think there is abetter way to do that. Can you use Link for it. I think that is more appropriate.

Reply:
I think you are right, I will fix it.